CMake/amiga_defs.cmake

```diff
@@ -3,7 +3,6 @@ set(UBSAN OFF)
 set(NONET ON)
 set(USE_SDL1 ON)
 set(SDL1_VIDEO_MODE_BPP 8)
-set(TTF_FONT_NAME \"LiberationSerif-Bold.ttf\")

```

Comment:
These defines are a good way to set the default value for the platform without littering `#ifdef` in the code

Reply:
It's important to have these as CMake defines because it lets package maintainers configure this default without modifying devilutionx code

Reply:
AJenbo said "If you make it specify an exact font we can get rid of the build time font options. So I guess that would be preferred." So IDK at this point.

Reply:
You changed my mind "It looks like its only the amiga that uses a different font. So would we just do an #ifdef that sets it for the amiga otherwise use charissilb?", sorry for the confusion :/

Reply:
I just put TTF_FONT_NAME back. It will also let somebody use any font they want by setting the TTF_FONT_NAME when building.

Reply:
Can you please put `TTF_FONT_DIR` back as well (as the default for when no `--ttf-dir` argument is provided)?

Some systems use a different font directory. E.g. batocera.linux is a Linux but has its custom paths for everything, etc.